tests/tests/runner.rs

```diff
@@ -219,6 +222,19 @@ async fn file_data_sources() {
         assert!(data_source.causality_region == causality_region.next());
         causality_region = causality_region.next();
     }
+
+    let stop_block = test_ptr(5);
+    let err = ctx.start_and_sync_to_error(stop_block).await;
+    let message = "entity type `IpfsFile1` is not on the 'entities' list for data source `File2`. \
```

Comment:
would it be enough to check if this is an isolation error somehow (left a suggestion about subtype but that might be lost). Maybe we could match an important part of the match like IpfsFile1 or somnething rather than the full message which also includes a backtrace. As it is it will be broken as soon as any change is made in this subgraph.

Reply:
If the test breaks when functionality changes, that's good! Gives us a chance to check that the change was intended. Even if its for the backtrace which is not directly unrelated to what's being tested.